bigbluebutton-html5/imports/ui/components/dropdown/component.jsx

```diff
@@ -186,7 +187,7 @@ class Dropdown extends Component {
       dropdownHide: this.handleHide,
     });
 
-    const showCloseBtn = (isOpen && keepOpen) || (isOpen && !keepOpen);
+    const showCloseBtn = (isOpen && keepOpen) || (isOpen && keepOpen === null);
```

Comment:
```suggestion
    const showCloseBtn = (isOpen && keepOpen) || (isOpen && keepOpen !== null);
```
Are you sure this is not flipped?

Reply:
Yeah, it needs to be  `===`